modules/nodes-pods-secrets-creating.adoc

```diff
@@ -120,6 +120,10 @@ spec:
           secretKeyRef: <1>
             name: test-secret
             key: username
+      from:
+        kind: ImageStreamTag
+        namespace: openshift
+        name: 'cli:latest'
```

Comment:
@rphillips Can you or someone PTAL? It appears that I needed to add the `from` stanza in order to get the BuildConfig to work. Thank you!